celery/bin/base.py

```diff
@@ -12,8 +12,7 @@
 
 from celery import VERSION_BANNER, Celery, maybe_patch_concurrency, signals
 from celery.exceptions import CDeprecationWarning, CPendingDeprecationWarning
-from celery.five import (PY2, getfullargspec, items, long_t,
-                         python_2_unicode_compatible, string, string_t,
+from celery.five import (getfullargspec, items, long_t, string, string_t,
```

Comment:
import from five still needed?

Reply:
This is removed in the CLI refactor anyway.

Reply:
ok cool